components/form/textInput/src/index.js

```diff
@@ -0,0 +1,59 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+const renderErrorMessage = error => (
+  error && (
+    <div className='sui-FormTextInput-errorMessage'>
+      <span className='sui-FormTextInput-errorMessageLabel'>{error}</span>
+    </div>
+  )
+)
+
+const FormTextInput = ({className, errorMessage, name, onChange, placeholder, value}) => (
+  <div className={cx('sui-FormTextInput', className)}>
+    <input
+      className='sui-FormTextInput-value'
+      name={name}
+      type='text'
+      placeholder={placeholder}
+      value={value}
+      onChange={onChange} />
+    {renderErrorMessage(errorMessage)}
```

Comment:
What do you think about checking if there is an error before to execute the function?
`error && renderErrorMessage(errorMessage)`

Reply:
Yes, it is better on that way. I will change it in next commit! Thanks! 👍

Reply:
Great!